packages/flutter_tools/lib/src/reporting/github_template.dart

```diff
@@ -102,7 +102,7 @@ class GitHubTemplateCreator {
           if (pluginParts.length != 2) {
             continue;
           }
-          description += pluginParts.first;
+          description += '${pluginParts.first}\n';
```

Comment:
nit: using a `StringBuffer` for this would probably make it a little clearer what was going on - you could then do something like `description.writeln(pluginParst.first);`

not-nit: we probably want to have a test for this somewhere, asserting that we generate the right template